tools/system_libs.py

```diff
@@ -1312,8 +1303,8 @@ def add_library(lib):
     # We need to build and link the library in
     add_library(lib)
 
+  add_library(system_libs_map['libcompiler_rt'])
```

Comment:
this may slow down fastcomp builds? may be worth measuring the impact on our total bot times perhaps.

Reply:
I'll take a look.  I'd be surprised since this is such a tiny library on fastcomp.. it only contains 4 symbols.

Reply:
Maybe I can revert this part.